src/app/shared/NoteReply.svelte

```diff
@@ -1,61 +1,54 @@
 <script lang="ts">
-  import {session, displayProfileByPubkey, tagReplyTo, tagPubkey} from "@welshman/app"
+  import {displayProfileByPubkey, session, tagPubkey, tagReplyTo} from "@welshman/app"
   import {ctx} from "@welshman/lib"
   import {getPubkeyTagValues, createEvent, uniqTags} from "@welshman/util"
-  import {writable, type Writable} from "svelte/store"
-  import {createEventDispatcher} from "svelte"
+  import {uniq, without} from "ramda"
   import {Editor} from "svelte-tiptap"
-  import {without, uniq} from "ramda"
-  import {slide} from "src/util/transition"
-  import AltColor from "src/partials/AltColor.svelte"
-  import Chip from "src/partials/Chip.svelte"
+  import {writable, type Writable} from "svelte/store"
   import {getEditorOptions} from "src/app/editor"
   import Compose from "src/app/shared/Compose.svelte"
-  import NsecWarning from "src/app/shared/NsecWarning.svelte"
   import NoteOptions from "src/app/shared/NoteOptions.svelte"
-  import {drafts} from "src/app/state"
-  import {publish, tagsFromContent, getClientTags, sign, userSettings} from "src/engine"
+  import NsecWarning from "src/app/shared/NsecWarning.svelte"
+  import {drafts, openReplies} from "src/app/state"
+  import {getClientTags, publish, sign, tagsFromContent, userSettings} from "src/engine"
+  import AltColor from "src/partials/AltColor.svelte"
+  import Chip from "src/partials/Chip.svelte"
+  import {slide} from "src/util/transition"
 
   export let parent
-  export let addDraftToContext
   export let showBorder = false
   export let forceOpen = false
 
-  const dispatch = createEventDispatcher()
+  let showOptions = false
+
   const nsecWarning = writable(null)
 
-  let container, loading
-  let isOpen = false
-  let showOptions = false
-  let mentions = []
-  let options = {warning: "", anonymous: false}
+  $: mentions = without(
+    [$session?.pubkey],
+    uniq([parent.pubkey, ...getPubkeyTagValues(parent.tags)]),
+  )
+
+  let loading
+  let opts = {warning: "", anonymous: false}
   let editorElement: HTMLElement
   let editor: Editor
   let editorLoading: Writable<boolean>
 
-  export const start = () => {
-    dispatch("start")
-
-    isOpen = true
-    mentions = without([$session.pubkey], uniq([parent.pubkey, ...getPubkeyTagValues(parent.tags)]))
-  }
-
   const bypassNsecWarning = () => {
     nsecWarning.set(null)
     send({skipNsecWarning: true})
   }
 
-  const openOptions = () => {
-    showOptions = true
-  }
-
   const closeOptions = () => {
     showOptions = false
   }
 
-  const setOptions = values => {
-    options = {...options, ...values}
-    showOptions = false
+  const openOptions = () => {
+    showOptions = true
+  }
+
+  const setOpts = e => {
+    opts = {...opts, ...e.detail}
```

Comment:
Looks like you've undone some of my changes, options are no longer working correctly on replies